### PR TITLE
RabbitMQ sample consumer 

### PR DESCRIPTION
**Pull request by Meruja**
And also user can override non-optional properties( uri, exchange.name and type of mapping).

### CODE REVIEW

File: modules/samples/sample-clients/rabbitmq-consumer/pom.xml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~  Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (ksdperera) [inline modules/samples/sample-clients/rabbitmq-consumer/pom.xml]: Please update the License.

File: modules/samples/sample-clients/rabbitmq-consumer/src/main/java/org/wso2/sp/sample/rabbitmq/consumer/RabbitMQReceiver.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+     * @param args no args need to be provided
+     */
+    public static void main(String[] args) {
+        log.info("Initialize rabbitmq receiver.");
+        SiddhiManager siddhiManager = new SiddhiManager();
+        String uri = args[0];
+        String exchange = args[1];
+        String type = args[2];
+        SiddhiAppRuntime siddhiAppRuntime = siddhiManager.createSiddhiAppRuntime(
+                "@App:name(\"PublishRabbitmqInXmlFormatTest\")\n" +
```
Comment (ksdperera) [inline modules/samples/sample-clients/rabbitmq-consumer/src/main/java/org/wso2/sp/sample/rabbitmq/consumer/RabbitMQReceiver.java]: Please change the siddhi app name with "RabbitMQSampleReceiver"

File: modules/samples/sample-clients/rabbitmq-consumer/build.xml [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+            <include name="slf4j.api_*.jar"/>
+            <include name="siddhi-*.jar"/>
+            <include name="disruptor_*.jar"/>
+            <include name="org.eclipse.osgi_*.jar"/>
+            <include name="quartz_*.jar"/>
+            <include name="antlr4-runtime_*.jar"/>
+            <include name="io.dropwizard.metrics.core_*.jar"/>
+            <include name="com.google.*.jar"/>
+            <include name="org.apache.ws.commons.axiom.axiom-api_*.jar"/>
+            <include name="jaxen_*.jar"/>
```
Comment (mohanvive) [inline modules/samples/sample-clients/rabbitmq-consumer/build.xml]: Recheck the dependencies that required.

File: modules/samples/sample-clients/rabbitmq-consumer/pom.xml [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>org.wso2.sp</groupId>
+        <artifactId>sample-clients</artifactId>
+        <version>4.0.0-beta2-SNAPSHOT</version>
```
Comment (mohanvive) [inline modules/samples/sample-clients/rabbitmq-consumer/pom.xml]: Parent pom version changed. Please fix.

File: modules/samples/sample-clients/rabbitmq-consumer/build.xml (hunk 3 above)
Comment (Meruja) [reply modules/samples/sample-clients/rabbitmq-consumer/build.xml]: checked

**Comment by Meruja** [comment]
Thank you

**Pull request merged**